doc/man7/provider-keymgmt.pod

```diff
@@ -238,10 +244,15 @@ progresses.
 OSSL_FUNC_keymgmt_gen_cleanup() should clean up and free the key object
 generation context I<genctx>
 
-At least one of OSSL_FUNC_keymgmt_new() and OSSL_FUNC_keymgmt_gen() are mandatory,
-as well as OSSL_FUNC_keymgmt_free().  Additionally, if OSSL_FUNC_keymgmt_gen() is
-present, OSSL_FUNC_keymgmt_gen_init() and OSSL_FUNC_keymgmt_gen_cleanup() must be
-present as well.
+OSSL_FUNC_keymgmt_load() is the third constructor, which creates a provider
+side key object based on a I<reference_sz> bytes large I<reference> that only
```

Comment:
This line need cleaning up..

Reply:
Er, I have no idea how

Reply:
Something like:
OSSL_FUNC_keymgmt_load() creates a provider side key object based on a I<reference>  object with a size of I<reference_sz> bytes, that only

Reply:
Huh...  to my ear, that sounded clumsier, but perhaps I'm listening to it with a Swedish mind, so I'll take your word for it.

Reply:
... or well, "is the third constructor, which" was clumsy, I agree.  I referred to `a I<reference>  object with a size of I<reference_sz> bytes` as opposed to `a I<reference_sz> bytes large I<reference>`.

Reply:
Anyway, done